data-model/src/main/scala/za/co/absa/enceladus/model/MappingTable.scala

```diff
@@ -15,39 +15,78 @@
 
 package za.co.absa.enceladus.model
 
-import java.time.ZonedDateTime
-import za.co.absa.enceladus.model.dataFrameFilter.DataFrameFilter
+import com.fasterxml.jackson.annotation.JsonIgnore
 import com.fasterxml.jackson.databind.node.ArrayNode
-import za.co.absa.enceladus.model.versionedModel.VersionedModel
-import za.co.absa.enceladus.model.backend.audit._
+import io.swagger.v3.oas.annotations.media.{ArraySchema, Schema => AosSchema}
 import za.co.absa.enceladus.model.backend.Reference
+import za.co.absa.enceladus.model.backend.audit._
+import za.co.absa.enceladus.model.dataFrameFilter.DataFrameFilter
+import za.co.absa.enceladus.model.versionedModel.VersionedModel
+
+import java.time.ZonedDateTime
+import scala.annotation.meta.field
+import scala.beans.BeanProperty
+
+case class MappingTable(
+  @(AosSchema@field)(example = "datasetA")
```

Comment:
Is that copy&paste typo? "datasetA" ==> "mappingTableA"

Reply:
Yes, of course, thanks. Fixed.